### ISSUE TITLE
State of the package

### ISSUE THREAD

**Issue by Gnimuc** [JuliaGPU/Vulkan.jl#2]
Hi @SimonDanisch, what's the current plan on this package? It looks like the development has been long suspended.

**Comment by SimonDanisch** [comment]
There is no plan :-D feel free to come up with one ;)

**Comment by serenity4** [comment]
Hi! I began to use the VulkanCore API to try to render some graphics, but it is the kind of API where crashes occur frequently when pointers are not handled properly. Therefore I started to define basic structures around VulkanCore types to solve pointer issues and also made some abstractions for convenience. So far, I have been doing it on my own repo, which is intended to be an app using Vulkan, not just the abstraction layer. If this project goes on, I am considering to merge that abstraction layer into Vulkan.jl. If anybody wishes to collaborate on this abstraction layer, or has already done some work, I guess it might be best to work with a dedicated repo (a fork of Vulkan.jl, or Vulkan.jl itself maybe?).
For now it seems to me that working with a single package is easier, in which case a split would happen later in the process. I am open to suggestions though!
